package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "silverstripe-subsites",
+  "description": "Builds the javascript for silverstripe subsites extension",
+  "main": "./client/src/boot/index.js",
+  "author": "You",
+  "engines": {
+    "node": ">=10.x"
+  },
+  "scripts": {
+    "build": "yarn && yarn lint && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "dev": "NODE_ENV=development webpack --progress",
+    "watch": "NODE_ENV=development webpack --watch --progress",
+    "css": "WEBPACK_CHILD=css npm run build",
+    "test": "jest",
+    "coverage": "jest --coverage",
+    "lock": "npm-shrinkwrap --dev",
```

Comment:
shouldn't need shrinkwrap for anything since we are using yarn?

Reply:
Probably correct. I'll remove it and see what happens.
Most amusingly it appears that shrinkwrap isn't listed as a dependency anyway 😂